dynamodb-stream-reader/src/main/java/elasticsearch/ElasticSearchRestClient.java

```diff
@@ -0,0 +1,124 @@
+package elasticsearch;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import nva.commons.utils.Environment;
+import org.apache.http.HttpHeaders;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+
+import static no.unit.nva.dynamodb.DynamoDBStreamHandler.ELASTICSEARCH_ENDPOINT_INDEX_KEY;
+
+public class ElasticSearchRestClient {
+
+    private static final Logger logger = LoggerFactory.getLogger(ElasticSearchRestClient.class);
+
+    private static final String ELASTICSEARCH_ENDPOINT_OPERATION = "_doc";
+    public static final String INITIAL_LOG_MESSAGE = "using Elasticsearch endpoint {} {} and index {}";
+    public static final String APPLICATION_JSON = "application/json";
```

Comment:
better create a Constants class so that you do not have to re-declare the constants. Also use the standard types that already exist `ContentType.APPLICATION_JSON.getMimeType()`.

Reply:
Done

Reply:
....and created Constans class, leaving class dependent messages behind...